tools/tree-diff

```diff
@@ -64,7 +64,10 @@ def content_diff(name, dir_fd1, dir_fd2, path, differences):
 
         if hash1 != hash2:
             props = differences.setdefault(path, {})
-            props["content"] = [hash1, hash2]
+            if name == "shadow" or name == "shadow-":
```

Comment:
Is this part of the diff still needed when the content is set to "null", "null" in the above "diff.json"?

Reply:
AFAIK, no....

Reply:
Just so I'm on the same page, you mean leave the section out entirely?

Reply:
it should be enough to set the hash values to `null` in the `diff.json` of the respective stage test. There shouldn't be any need to modify `tools/tree-diff`.

Reply:
Ah right. Will fix